osu.Game/Online/Chat/LinkWarnMode.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+namespace osu.Game.Online.Chat
```

Comment:
I guess this is a foregone conclusion but this really doesn't fit in this namespace anymore, the class has definitively ceased to be chat-specific. Probably needs to be moved somewhere more general in a follow-up.

Reply:
yeah, nowhere better to put it though.